### DISCUSSION TITLE
why were conditions on the dynamic attributes `forbidden` and `mandatory` removed?

### DISCUSSION THREAD

**Discussion by karel-debrabandere-bosa-fgov-be** [OP-TED/eForms-SDK#223] [Q&A] [answered]
Ref. commit: https://github.com/OP-TED/eForms-SDK/blob/d27a7bfe31ee5ba930bc26fa7d428233f0c71d05/fields/fields.json

As far as I can see, all condition constraints on all dynamic attributes (except `assert`) have been removed from the fields.json. Is this correct?

**Comment by bertrand-lorentz** [comment, marked as answer]
As indicated in the [release notes for SDK 1.3.0](https://github.com/OP-TED/eForms-SDK/releases/tag/1.3.0), we have removed those constraints in order to make it initially easier to fill in notices for end users of applications like eNotices2 but also making them more responsible for providing coherent data.
The rules will be added back in future SDKs, allowing users and systems to integrate them progressively and to improve data quality, while also improving the user experience of filling in the notices.
